db/main/migrate/20180329000000_create_github_installations.rb

```diff
@@ -0,0 +1,32 @@
+class CreateGithubInstallations < ActiveRecord::Migration
+  PSQL = ActiveRecord::Base.connection.select_value('SELECT version()')
+
+  def self.up
+    create_table :github_installations do |t|
```

Comment:
we also need a `github_installation_id`, which GitHub sends us in the `InstallationEvent` (this is needed for generating the tokens)